alpaca/version.go

```diff
@@ -0,0 +1,38 @@
+package alpaca
+
+import (
+	"runtime/debug"
+	"strings"
+	"sync"
+)
+
+const repoName = "github.com/alpacahq/alpaca-trade-api-go"
+
+var (
+	once            = sync.Once{}
+	encodedVersions string
+)
+
+// GetVersion returns running go version and alpaca-trade-api-go version
+func GetVersion() string {
+	once.Do(func() {
+		var (
+			goVersion     string
+			moduleVersion string
+		)
+
+		buildInfo, found := debug.ReadBuildInfo()
+		if !found {
+			return
+		}
+		goVersion = buildInfo.GoVersion
```

Comment:
minor: I would stick to `runtime.Version()` instead of the build version, I think it is more helpful. Moreover, if build info is not available we can still return the header!

Reply:
Addressed: https://github.com/alpacahq/alpaca-trade-api-go/pull/254/commits/783440c543b38222ed2b864afaa78626d5f9968e